bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/GmosSDtaXRuleTest.scala

```diff
@@ -56,31 +56,45 @@ class GmosSDtaXRuleTest extends AbstractRuleTest {
     rule.check(elems).getProblems.asScala.toList
   }
 
-  private def expectSuccess(): Unit = {
-//    println(runCheck.mkString(",\n"))
-    assertTrue(runCheck.isEmpty)
+  private def expectSuccess(hasId: Set[String]): Unit =
+    assertTrue(!runCheck.exists(p => hasId(p.getId)))
+
+  private def expectSuccessBinY(): Unit =
+    expectSuccess(Set(
+      GmosRule.GMOS_S_DTA_X_RULE_Y1_ID,
+      GmosRule.GMOS_S_DTA_X_RULE_Y2_4_ID)
+    )
+
+  private def expectSuccessBinMultiple(): Unit = {
+    assertTrue(!runCheck.exists { p =>
+      GmosRule.DTA_X_Y_MULTIPLE_BINNING_RULE_ID == p.getId()
+    })
```

Comment:
It looks like this could be just a call to `expectSuccess` with a singleton `Set` containing the one rule.